src/abstractdataframe/abstractdataframe.jl

```diff
@@ -438,8 +438,7 @@ completecases(df)
 ```
 
 """
-function completecases(df::AbstractDataFrame)
-    ## Returns a Vector{Bool} of indexes of complete cases (rows with no NA's).
+function completecases(df::AbstractDataFrame)::AbstractVector{Bool}
```

Comment:
Alright, I decided to go with @nalimilan's approach since it would at least cause an earlier failure conditions if for some reason the output can't be converted to `AbstractVector{Bool}` and might have better performance in some situations.